runtime-decompiler/src/main/java/org/jrd/frontend/PluginMangerFrame/FileSelectorPanel.java

```diff
@@ -21,8 +23,13 @@
         textField.setPreferredSize(new Dimension(0, 32));
         this.jLabel = new JLabel(label);
         this.browseButton = new JButton(ButtonLabel);
+
         this.chooser = new JFileChooser();
-        chooser.setCurrentDirectory(new File(System.getProperty("user.home")));
+        File dir = new File(Directories.getPluginDirectory());
```

Comment:
please extract the duplicated code to method

Reply:
Duplicated where?

Reply:
File dir = new File(Directories.getPluginDirectory());
        while(!dir.exists()){
            dir = dir.getParentFile();
        }


nd few lines below
  while(!dir.exists()){
            dir = dir.getParentFile();
        }
        chooser.setCurrentDirectory(dir);


in  https://github.com/pmikova/java-runtime-decompiler/pull/51/commits/19e65813e0f5d3e63910be3d64c27b2d3be56c71

Reply:
I guess you mean that the same loop is used in both frontend/PluginMangerFrame/FileSelectorPanel.java and frontend/PluginMangerFrame/FileSelectorArrayRow.java, right?